house/urls.py

```diff
@@ -1,13 +1,13 @@
 from django.urls import path
-
 from house import views
 
 app_name = 'house'
 urlpatterns = [
     path('', views.home_page, name='home'),
     path('global/', views.global_page, name='global_page'),
-    path('login/', views.house_login, name='house_login'),
+    path('login', views.house_login, name='house_login'),
```

Comment:
it is supposed to be login/ 

Also, I can't log in to see what the page looks like because it seems to be a problem in the house creation using the admin panel:
![image](https://user-images.githubusercontent.com/57755873/166137946-617204bb-406a-4a39-9c9e-cffd6a829118.png)
I cant select  values for country/city

Reply:
This is a database issue. you can try to delete the dbsqulite file and run migrate

Reply:
fixed the login url 
I didn't change anything on the house model. 
Rony is suppose to open a PR for creating house, if there is a problem he will fix it 
@ronyyosef @adiabramovitch